### PR TITLE
Update edit_distance.py

### PR DESCRIPTION
**Pull request by pedram-mohajer**
### Describe your change:

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: strings/edit_distance.py [hunk 1]
```diff
@@ -1,32 +1,48 @@
 def edit_distance(source: str, target: str) -> int:
     """
-    Edit distance algorithm is a string metric, i.e., it is a way of quantifying how
-    dissimilar two strings are to one another. It is measured by counting the minimum
-    number of operations required to transform one string into another.
+    Calculate the edit distance between two strings.
```
Comment (imSanko) [inline strings/edit_distance.py:3]: Can we change the documentation as we'll the code too at the same time?

Comment (pedram-mohajer) [reply strings/edit_distance.py:3]: Is there any rule about it?

Comment (imSanko) [reply strings/edit_distance.py:3]: `You need to opt` for the option on which you're working with >_

Comment (pedram-mohajer) [reply strings/edit_distance.py:3]: Got it, I'll close this and pull another one.

**Pull request closed without merging**